finance/community-updates/2022-10.txt

```diff
@@ -0,0 +1,94 @@
+Dear Astropy Community,
+
+The Finance Committee would like to give you some updates on our activities
+over the several months.
+
+
+# Cycle III Funding Results
+
+We recently completed the proposal process for allocating the next round of
+funding for projects and other major expenses. This is the first time we have
+been able to allocate money from our NASA ROSES-2020 grant as well as our
+grant from the Moore foundation.
+
+A total of 22 proposals were received, of which 20 were allocated funds.
+Although our budget didn't quite stretch to supporting all projects at their
+full requested level, every approved proposal receives at least their minimum
+funding request, and — in many cases — substantially more.
+
+The full list of proposals funded in this process is available at [1]. Each
+proposal will be allocated a dedicated Finance Committee member who will guide
+project team through the process of getting started and claiming funds.
+
+We're excited to be able to fund so many amazing projects! Thanks to those
+who submitted proposals and to everybody who took part in the discussion and
+voting process, and many congratulations to the successful project teams!
+
+The Finance Committee welcomes feedback on the funding process. Please feel
+free to reach out to us — see the contact details below — with your thoughts
+and suggestions for how it might be improved in the future.
+
+[1] https://github.com/astropy/astropy-project/pulls?q=label%3A%22funding-approved+💵%22+label%3A%22cycle+3%EF%B8%8F⃣%22+type%3Apr
+
+
+# Keeping Track of Work in Progress
+
+The Finance Committee uses GitHub issues to keep track of all the various ways
+in which we are allocating funds. If you would like to know where the
+Project's funds are being spent, check out the “finance” label in the
+astropy-project issue tracker [2].
+
+For previous funding rounds, we've used a single issue to keep track of all
+the various projects [3]. During Cycle III, we plan to use a single issue per
+project; watch out for these appearing over the next few days.
+
+[2] https://github.com/astropy/astropy-project/labels/finance%20%3Amoneybag%3A
+[3] https://github.com/astropy/astropy-project/issues/199/
+
+
+# Open Collective
+
+Over the spring and summer of this year, NumFOCUS — the organization that
+manages our accounts — switched to using Open Collective [4] for handling
+budgeting and invoicing. This new system is a substantial step forward — it's
+easier to use and more transparent than its predecessor.
+
+All invoices must now be submitted through Open Collective. This process is
+straightforward, but it is important that you know which of our various
```

Comment:
```suggestion
straightforward (different but generally easier than the previous "Rocket" system), but it is important that you know which of our various
```

Reply:
I feel like we basically said “it's easier than the old system” in the previous paragraph. I'll add a mention of Rocket to that, but then I think that this is redundant.